app/assets/javascripts/school_administrator_dashboard/students_table.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+
+export default React.createClass({
+  displayName: 'StudentsTable',
+
+  propTypes: {
+    rows: React.PropTypes.array.isRequired
+  },
+
+  render: function() {
+    return(
+      <div className= 'StudentsList'>
+        <table className='students-list' style={{width: '100%'}}>
+          <thead>
+            <tr>
+              <td>
+                {"Last Name"}
```

Comment:
@edavidsonsawyer Small thing here but no need to use the curly brackets if you're just returning a string.